dwhdelft.nl/pages/actieplan.vue

```diff
@@ -0,0 +1,141 @@
+        - 'The municipality has engaged with associations in the Sports Agreement to publish their LGBTQ+ vision on their websites as a sign of inclusivity (Action: Municipality of Delft)'
+        - 'The municipality will propose to the Positive Sports Climate working group (part of the Sports Agreement) to include LGBTQ+ topics as a recurring agenda item (Action: Municipality of Delft, Haaglanden Beweegt)'
+        - 'Share information on www.sportenindelft.nl about gender-neutral/mixed teams and competitions (Action: Delft voor Elkaar / Haaglanden Beweegt)'
+    - name: Well-being
+      tasks:
+        - 'The Municipality of Delft and the Customer Contact Center will conduct a client journey and revise protocols based on the findings (Action: Municipality of Delft)'
+        - 'Youth psychosocial services in Delft (e.g., the Youth Information Point) are equipped with LGBTQ+ expertise and can refer youth to local events and support networks (Action: DWH)'
+        - 'Pink 50+ and the Municipality of Delft have met with care institutions to create LGBTQ+-friendly locations and publish their policies online (Action: Pink 50+, Municipality of Delft)'
+        - 'The Pink Salon at Delfshove as a meeting place for LGBTQ+ elders will continue.'
+    - name: City
+      tasks:
+        - 'The municipality explores where and how to omit gender registration on municipal forms (Action: Municipality of Delft)'
+        - 'The municipality examines the possibilities for gender-neutral toilets in public areas of the municipal office (Action: Municipality of Delft)'
+        - 'On Coming Out Day, the alderman and LGBTQ+ policy officer raise the pride flag with local LGBTQ+ advocates (Action: Municipality of Delft, DWH)'
+        - 'The municipality maintains LGBTQ+ symbols in the city, such as the rainbow bridge and electricity boxes (Action: Municipality of Delft)'
+        - 'Each year before Purple Friday, the municipality holds an evaluation meeting with stakeholders to assess progress on the Rainbow Action Plan (Action: Municipality of Delft)'
+        - 'The municipality actively collaborates with other rainbow municipalities in the region (Action: Municipality of Delft)'
+        - 'The municipality makes its LGBTQ+ policy and services for the target group visible on its website (Action: Municipality of Delft)'
+    - name: Neighborhoods and Faith Communities
+      tasks:
+        - 'DWH and the municipality engage in conversations in neighborhoods and with various faith communities about LGBTQ+ (Action: DWH)'
+        - 'DWH collaborates with organizations to increase LGBTQ+ visibility and organize activities in neighborhoods outside the city center (Action: DWH)'
+  document-title: Documents
+</i18n>
+
+<script setup>
+const { t } = useT()
+
+const localFiles = [
+  {
+    name: 'xRegenboog Actie Plan 2023 - 2026',
```

Comment:
```suggestion
    name: 'Regenboog Actie Plan 2023 - 2026',
```

Reply:
For some reason valt de eerste letter weg in het component, dit was mijn quick fix